emitters/kiibohd/kiibohd.py

```diff
@@ -719,7 +720,7 @@ def process( self ):
 
 		## Default Layer and Default Layer Scan Map ##
 		self.fill_dict['DefaultLayerTriggerList'] = ""
-		self.fill_dict['DefaultLayerScanMap'] = "const nat_ptr_t *default_scanMap[] = {\n"
+		self.fill_dict['DefaultLayerScanMap'] = "const nat_ptr_t *default_scanMap[] = { \n"
```

Comment:
Not sure why this is in the diff to be honest.
This is the fix for peculiar python formatting.